client/my-sites/stats/stats-date-picker/index.jsx

```diff
@@ -12,8 +13,36 @@ class StatsDatePicker extends Component {
 		] ),
 		period: PropTypes.string.isRequired,
 		summary: PropTypes.bool,
+		query: PropTypes.object,
 	};
 
+	dateForSummarize() {
+		const { query, moment, translate } = this.props;
+		const localizedDate = moment();
+		let formattedDate;
+
+		switch ( query.num ) {
+			case '-1':
+				formattedDate = translate( 'All Time' );
+				break;
+
+			default:
+				formattedDate = translate(
+					'%(number)s days ending %(endDate)s (Summarized)',
```

Comment:
Alternate string suggestion: 
* `translate( 'Referrers for %1$s days ending %2$s (Summarized)' )`&mdash; translations: **48**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/stats/all-time-summary)